frontend/ui/message/simpletcpserver.lua

```diff
@@ -23,7 +25,11 @@ function SimpleTCPServer:new(o)
 end
 
 function SimpleTCPServer:start()
-    self.server = socket.bind(self.host, self.port)
+    local server, err = socket.bind(self.host, self.port)
+    if not server then
+        return false, (err and T(_("Failed to bind socket: %1"), err) or "Failed to bind socket")
```

Comment:
One translated and the other not ?
`T(_("Failed to bind socket: %1"), err or "n/a")` or something like that ?

s/found/bound/ in your PR title :)

Reply:
```suggestion
        return false, (err and T(_("Failed to bind socket: %1"), err) or _("Failed to bind socket"))
```

Reply:
Whoops, thanks.